data/uhc_pack/functions/starting/starting_health.mcfunction

```diff
@@ -0,0 +1,24 @@
+#
+# Set starting health according to setting
+#
+
+execute if score UHC uhcStHealth matches 12 run effect give @a[tag=playing] minecraft:health_boost 999999 0 true
+execute if score UHC uhcStHealth matches 12 run effect give @a[tag=playing] minecraft:instant_health 1 0
+execute if score UHC uhcStHealth matches 14 run effect give @a[tag=playing] minecraft:health_boost 999999 1 true
+execute if score UHC uhcStHealth matches 14 run effect give @a[tag=playing] minecraft:instant_health 1 1
+execute if score UHC uhcStHealth matches 16 run effect give @a[tag=playing] minecraft:health_boost 999999 2 true
+execute if score UHC uhcStHealth matches 16 run effect give @a[tag=playing] minecraft:instant_health 1 2
+execute if score UHC uhcStHealth matches 18 run effect give @a[tag=playing] minecraft:health_boost 999999 3 true
+execute if score UHC uhcStHealth matches 18 run effect give @a[tag=playing] minecraft:instant_health 1 2
+execute if score UHC uhcStHealth matches 20 run effect give @a[tag=playing] minecraft:health_boost 999999 4 true
+execute if score UHC uhcStHealth matches 20 run effect give @a[tag=playing] minecraft:instant_health 1 3
+execute if score UHC uhcStHealth matches 22 run effect give @a[tag=playing] minecraft:health_boost 999999 5 true
+execute if score UHC uhcStHealth matches 22 run effect give @a[tag=playing] minecraft:instant_health 1 3
+execute if score UHC uhcStHealth matches 24 run effect give @a[tag=playing] minecraft:health_boost 999999 6 true
+execute if score UHC uhcStHealth matches 24 run effect give @a[tag=playing] minecraft:instant_health 1 3
+execute if score UHC uhcStHealth matches 26 run effect give @a[tag=playing] minecraft:health_boost 999999 7 true
+execute if score UHC uhcStHealth matches 26 run effect give @a[tag=playing] minecraft:instant_health 1 3
+execute if score UHC uhcStHealth matches 28 run effect give @a[tag=playing] minecraft:health_boost 999999 8 true
+execute if score UHC uhcStHealth matches 28 run effect give @a[tag=playing] minecraft:instant_health 1 3
```

Comment:
It seems that if you set the health to 28 hearts, 2 of them will not be healed. I believe this will need to be instant health level 4 instead